modules/nvidia-gpu-aws-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+//
+//  * installing/installing-nvidia-gpu-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-adding-a-gpu-node_{context}"]
+= Adding a GPU node
+
+The following procedure shows you how to
+copy and modify a default worker MachineSet configuration to create a
+GPU-enabled MachineSet and Machines for the AWS EC2 cloud provider.
+
+.Procedure
+
+. View the existing nodes, Machines, and MachineSets. Note that
+each node is an instance of a machine definition with a specific AWS region and OpenShift role.
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME STATUS ROLES AGE VERSION
+
+ip-10-0-131-156.us-east-2.compute.internal Ready master 21m v1.14.6+c07e432da
+
+ip-10-0-132-241.us-east-2.compute.internal Ready worker 16m v1.14.6+c07e432da
+
+ip-10-0-144-128.us-east-2.compute.internal Ready master 21m v1.14.6+c07e432da
+
+ip-10-0-151-24.us-east-2.compute.internal Ready worker 15m v1.14.6+c07e432da
+
+ip-10-0-166-12.us-east-2.compute.internal Ready worker 16m v1.14.6+c07e432da
+
+ip-10-0-173-34.us-east-2.compute.internal Ready master 21m v1.14.6+c07e432da
+----
+
+. View the Machines and MachineSets that exist in
+the openshift-machine-api namespace. Each worker machine set is associated with
+a different availability zone within the AWS region. The installer automatically
+load balances workers across availability zones.
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
++
+[source,terminal]
+----
+AME DESIRED CURRENT READY AVAILABLE AGE
+
+openshift-blog-txxtf-worker-us-east-2a 1 1 1 1 22m
+
+openshift-blog-txxtf-worker-us-east-2b 1 1 1 1 22m
+
+openshift-blog-txxtf-worker-us-east-2c 1 1 1 1 22m
+----
+
+. At this time, there is only one worker machine per MachineSet, though a MachineSet
+could be scaled to add a node in a particular region and zone.
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api | grep worker
+----
++
+.Example output
++
+[source,terminal]
+----
+openshift-blog-txxtf-worker-us-east-2a-8grnb running m4.large us-east-2 us-east-2a 51m
+
+openshift-blog-txxtf-worker-us-east-2b-vw4ph running m4.large us-east-2 us-east-2b 51m
+
+openshift-blog-txxtf-worker-us-east-2c-qpd4q running m4.large us-east-2 us-east-2c 51m
+----
+
+. Make a copy of one of the existing worker
+MachineSet definitions and output the result to a JSON file.
+This will be the basis for the GPU-enabled worker machine set definition.
++
+[source,terminal]
+----
+$ oc get machineset openshift-blog-txxtf-worker-us-east-2a -n openshift-machine-api -o json >
+----
```

Comment:
Need to specify a file name for redirected machineset json output file after the ">".

Reply:
Added <output_file.json>